src/screens/WhotoFollow/elements/UserInfo.js

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
 import {Image, StyleSheet, Text, View} from 'react-native';
 import PropTypes from 'prop-types';
-import {normalize, normalizeFontSize} from '../../../utils/fonts';
+import {fonts, normalize, normalizeFontSize} from '../../../utils/fonts';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement has been updated to include `fonts` from the `../../../utils/fonts` module. This change is likely made to use the new font family in the component, replacing the previous hardcoded 'Poppins' font with a more maintainable and consistent approach.

```diff
- import {normalize, normalizeFontSize} from '../../../utils/fonts';
+ import {fonts, normalize, normalizeFontSize} from '../../../utils/fonts';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->